### PR TITLE
docs: incorrectly encoded url in example

### PR DESCRIPTION
**Pull request by stefan-guggisberg**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by kptdobe** [comment]
Are you sure about that ? Usually, xml does not like the `&` character...

**Comment by stefan-guggisberg** [comment]
> Are you sure about that ? Usually, xml does not like the & character...

No, I am not sure. But I think that Fastly doesn't care whether it's xml or not when it is processing the `<esi include` statement. I have a hunch that Fastly doesn't decode the `&amp;` when processing the ESI.

**Comment by trieloff** [comment]
The feeds in `theblog` seem to work (at least the breakage seems unrelated to the ampersands), but it would be worth testing there to make sure both encoding variants work just as well. I remember some iterations until I found a working set of encodings.

File: docs/feed.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ You need a data source which lists page URLs, page titles, and page update times
   <title>Example Feed</title>
   <link href="https://example.org/"/>
   <id>https://example.org/</id>
-  <esi:include src="/cgi-bin/feed.xml?src=/en/query-index.json%3Flimit%3D10&amp;id=path&amp;title=title&amp;updated=date">
+  <esi:include src="/cgi-bin/feed.xml?src=/en/query-index.json%3Flimit%3D10&id=path&title=title&updated=date">
```
Comment (tripodsan) [inline docs/feed.md:31]: I don't think this change necessary. the `&amp;` is the correct escaping to use in attributes.

**Review by tripodsan — CHANGES_REQUESTED** [review]

**Comment by stefan-guggisberg** [comment]
> The feeds in theblog seem to work (at least the breakage seems unrelated to the ampersands), but it would be worth testing there to make sure both encoding variants work just as well.

I have tested both encodings and both work. Closing this PR.

**Pull request closed without merging**